### PR TITLE
[NFC] clang-format codebase

### PR DESCRIPTION
**Pull request by gchatelet**

### CODE REVIEW

**Comment by Mizux** [comment]
checked locally with:
```sh
find include scripts src ndk_compat cmake test -type f | grep -v "\.txt$\|\.sh$\|\.in$\|\.md$"   | xargs clang-format --Werror --style=file --dry-run
```
and
```sh
find include src test ndk_compat \( -iname "*.c" -or -iname "*.h" -or -iname "*.cc" \) | xargs clang-format --Werror --style=file --dry-run
```

**Review by Mizux — APPROVED** [review]
LGTM

File: src/cpuinfo_aarch64.c [hunk 1]
```diff
@@ -75,56 +75,143 @@ DECLARE_SETTER_AND_GETTER(Aarch64Features, rng)
...
-  [AARCH64_SVEF32MM] = {{0, AARCH64_HWCAP2_SVEF32MM}, "svef32mm", &set_svef32mm, &get_svef32mm},
-  [AARCH64_SVEF64MM] = {{0, AARCH64_HWCAP2_SVEF64MM}, "svef64mm", &set_svef64mm, &get_svef64mm},
-  [AARCH64_SVEBF16] = {{0, AARCH64_HWCAP2_SVEBF16}, "svebf16", &set_svebf16, &get_svebf16},
-  [AARCH64_I8MM] = {{0, AARCH64_HWCAP2_I8MM}, "i8mm", &set_i8mm, &get_i8mm},
-  [AARCH64_BF16] = {{0, AARCH64_HWCAP2_BF16}, "bf16", &set_bf16, &get_bf16},
-  [AARCH64_DGH] = {{0, AARCH64_HWCAP2_DGH}, "dgh", &set_dgh, &get_dgh},
-  [AARCH64_RNG] = {{0, AARCH64_HWCAP2_RNG}, "rng", &set_rng, &get_rng},
-  [AARCH64_BTI] = {{0, AARCH64_HWCAP2_BTI}, "bti", &set_bti, &get_bti},
+    [AARCH64_FP] = {{AARCH64_HWCAP_FP, 0}, "fp", &set_fp, &get_fp},
+    [AARCH64_ASIMD] = {{AARCH64_HWCAP_ASIMD, 0},
```
Comment (Mizux) [inline src/cpuinfo_aarch64.c:79]: no way to fix this indent, one liner was better IMHO...

**Comment by gchatelet** [comment]
@Mizux `clang-format Check / clang-format (pull_request) Successful in 16s` :+1:

**Pull request merged**